src/main/java/picard/vcf/UpdateVcfSequenceDictionary.java

```diff
@@ -89,7 +91,15 @@ protected int doWork() {
         if (CREATE_INDEX)
             builder.setOption(Options.INDEX_ON_THE_FLY);
 
-        final VariantContextWriter vcfWriter = builder.setOutputFile(OUTPUT).build();
+        // handle stdout
+        if ( OUTPUT.getPath().equals(STD_OUT) ) {
```

Comment:
That's cleaner but how can the stdout be redirected to a file for validation by a test? With `System.out`, it's easy by using `System.setOut(PrintStream out)`.